v2/coord/internal/nettest/layouts.go

```diff
@@ -38,7 +37,7 @@ func LinearTopology(n int, clk clock.Clock) (*Topology, []*Node, error) {
 		nodes[i] = &Node{
 			NodeInfo:     ai,
 			Router:       NewRouter(ai.ID, top),
-			RoutingTable: simplert.New[key.Key256, kad.NodeID[key.Key256]](kadt.PeerID(ai.ID), 20),
+			RoutingTable: simplert.New[key.Key256, kadt.PeerID](kadt.PeerID(ai.ID), 20),
```

Comment:
Sometimes we're using `key.Key256` and sometimes the alias `KadKey` in `coord`. It would be great if we could be consistent and use either everywhere.

1. use key.Key256 everywhere
2. move coord.KadKey to, e.g., kadt.Key and use it from there
3. (make the coordinator generic on the key type and use `K` everywhere)

Intuitively I would have done 2 🤷‍♂️ but don't have a strong opinion.

Reply:
Agree on 2. I just missed this one. Will do another iteration of search and replace :D